README.md

```diff
@@ -136,7 +144,7 @@ Kor provides various subcommands to identify and list unused resources. The avai
   -e, --exclude-namespaces strings   Namespaces to be excluded, split by commas. Example: --exclude-namespaces ns1,ns2,ns3. If --include-namespaces is set, --exclude-namespaces will be ignored.
       --group-by string              Group output by (namespace, resource) (default "namespace")
   -h, --help                         help for kor
-      --include-labels string        Selector to filter in, Example: --include-labels key1=value1.(currently supports one label)
+      --include-labels string        Selector to filter in, Example: --include-labels key1=value1. (currently supports one label)
```

Comment:
This part is a copy of the CMD output, please also edit `cmd/root.go` to adjust the flag rephrasing in the actual output.

Also it's a good opportunity to re-copy the output to avoid drifts from the current release.

Reply:
Done